training-node/app/services/user.js

```diff
@@ -0,0 +1,14 @@
+const User = require('../models').users,
+  errors = require('../errors'),
+  logger = require('../logger'),
+  Sequelize = require('sequelize');
+
+exports.create = user =>
+  User.create(user)
+    .catch(Sequelize.ValidationError, e => {
```

Comment:
Why two catches? with one its ok, and inside the function you can check the error